SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -200,6 +203,38 @@ bool FSpatialGDKEditorModule::ShouldPackageMobileCommandLineArgs() const
 	return GetDefault<USpatialGDKEditorSettings>()->bPackageMobileCommandLineArgs;
 }
 
+uint32 GetPIEServerWorkers()
```

Comment:
do we still need that?
I think the only place that it got used inside the GDK was the editorworkercontroller (which is now gone) and you removed all instance of it in the engine, right?

Reply:
Yep, the intention was just to keep this one in its own function so `ForEveryServerWorker` looks cleaner. Could also move this into an anonymous namespace up top to make it clearer.